### ISSUE TITLE
Incompatible with newer Geyser versions

### ISSUE THREAD

**Issue by ChiboYen** [kejonaMC/GeyserUpdater#102]
```
[18:49:54 ERROR]: An error occurred while running this command.
java.util.concurrent.CompletionException: java.lang.RuntimeException: Unable to invoke command geyserupdate for com.velocitypowered.proxy.console.VelocityConsole@2ed8629e
        at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315) ~[?:?]
        at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320) ~[?:?]
        at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:649) ~[?:?]
        at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
        at java.lang.Thread.run(Thread.java:840) [?:?]
Caused by: java.lang.RuntimeException: Unable to invoke command geyserupdate for com.velocitypowered.proxy.console.VelocityConsole@2ed8629e
        at com.velocitypowered.proxy.command.VelocityCommandManager.executeImmediately0(VelocityCommandManager.java:239) ~[velocity.jar:3.2.0-SNAPSHOT (git-81b183ac-b276)]
        at com.velocitypowered.proxy.command.VelocityCommandManager.lambda$executeAsync$0(VelocityCommandManager.java:253) ~[velocity.jar:3.2.0-SNAPSHOT (git-81b183ac-b276)]
        at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
        ... 4 more
Caused by: java.lang.NoSuchMethodError: 'java.io.InputStream org.geysermc.geyser.GeyserBootstrap.getResource(java.lang.String)'
        at com.projectg.geyserupdater.common.util.GeyserProperties.getGeyserGitPropertiesValue(GeyserProperties.java:41) ~[?:?]
        at com.projectg.geyserupdater.common.util.GeyserProperties.isLatestBuild(GeyserProperties.java:25) ~[?:?]
        at com.projectg.geyserupdater.velocity.command.GeyserUpdateCommand.execute(GeyserUpdateCommand.java:21) ~[?:?]
        at com.projectg.geyserupdater.velocity.command.GeyserUpdateCommand.execute(GeyserUpdateCommand.java:13) ~[?:?]
        at com.velocitypowered.proxy.command.registrar.InvocableCommandRegistrar.lambda$createLiteral$1(InvocableCommandRegistrar.java:81) ~[velocity.jar:3.2.0-SNAPSHOT (git-81b183ac-b276)]
        at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:262) ~[velocity.jar:3.2.0-SNAPSHOT (git-81b183ac-b276)]
        at com.velocitypowered.proxy.command.VelocityCommandManager.executeImmediately0(VelocityCommandManager.java:226) ~[velocity.jar:3.2.0-SNAPSHOT (git-81b183ac-b276)]
        at com.velocitypowered.proxy.command.VelocityCommandManager.lambda$executeAsync$0(VelocityCommandManager.java:253) ~[velocity.jar:3.2.0-SNAPSHOT (git-81b183ac-b276)]
        at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
        ... 4 more
```
The plugin won't load on `Velocity 3.2.0-SNAPSHOT (git-81b183ac-b276)`

**Comment by Konicai** [comment]
Thanks. I'll try to get this fixed later tonight

**Comment by TBYT** [comment]
i also experience this, but i'm not using velocity.

**Comment by Konicai** [comment]
Resolved in 4ff453f74de76d95b9d5e6663c61f7b5bac7aa17
